backend/module/chaos/serverexperimentation/xds/xds_cache.go

```diff
@@ -132,6 +133,16 @@ func setSnapshot(resourceMap map[gcpTypes.ResponseType][]gcpTypes.ResourceWithTt
 		return err
 	}
 
+	if isSnapshotEmpty {
+		scope.Tagged(map[string]string{
+			"cluster": cluster,
+		}).Counter("active_faults").Inc(-1)
+	} else {
+		scope.Tagged(map[string]string{
+			"cluster": cluster,
+		}).Counter("active_faults").Inc(1)
+	}
```

Comment:
from what i can tell, a gauge would work better here than a counter.

Reply:
Yeah a gauge seems like a better fit

Reply:
shoot. I had a Guage and then while looking for tagging did copy/paste and missed replacing it. thanks for catching